src/javascript/app/common/attach_dom/popup.js

```diff
@@ -15,7 +15,7 @@ const showPopup = (options) => {
     }
     if (cache[options.url]) {
         callback(options);
-    } else {
+    } else if (options.url) {
```

Comment:
how could `options.url` trigger since its commented out ?

Reply:
we need this condition because we don't want the code below to get executed if there is no options.url (e.g. when it's commented out like in our case).